internal/explorer.point/public/index.zhtml

```diff
@@ -4,10 +4,6 @@
         <meta charset="utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <meta name="theme-color" content="#000000" />
-        <meta
```

Comment:
Why do we have to delete it?

Reply:
It was a useless tag with non-standard `name` property and placeholder content: https://www.w3schools.com/tags/att_meta_name.asp